cpp/src/arrow/buffer-test.cc

```diff
@@ -46,6 +46,14 @@ TEST_F(TestBuffer, IsMutableFlag) {
   ASSERT_TRUE(pbuf.is_mutable());
 }
 
+TEST_F(TestBuffer, FromStdString) {
```

Comment:
Does this need to be `TEST_F` or can it just be `TEST`? It doesn't appear to be using any of the members of `TestBuffer`.

Reply:
I removed the fixture and made everything `TEST` since none of them were using it